client/app/scripts/reducers/root.js

```diff
@@ -721,6 +721,12 @@ export function rootReducer(state = initialState, action) {
       return state.set('showingTroubleshootingMenu', !state.get('showingTroubleshootingMenu'));
     }
 
+    case ActionTypes.CHANGE_INSTANCE: {
+      state = state.set('selectedNodeId', null);
```

Comment:
This line is also redundant now as `closeAllNodeDetails` takes care of it. Otherwise LGTM!